stream-platform/src/main/java/io/camunda/zeebe/stream/impl/NoSuchProcessorException.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.stream.impl;
+
+import io.camunda.zeebe.stream.api.records.TypedRecord;
+import io.camunda.zeebe.util.exception.UnrecoverableException;
+
+/**
+ * Exception type thrown by the {@link ProcessingStateMachine} when encountering a command that is
+ * not accepted by any of the registered processors.
```

Comment:
💭 Since we don't expect ever a command that is not processed? 🤔 Might there be any case for this, the backup processor is next to the engine right in the same state machine so this will not cause any issue I guess?

Reply:
Yeah, this assumes that all commands have a matching processor.
Based on that all tests are passing and the engine registers processors for all commands + we register one processor for Checkpoint records.